operations/network/management_network/README.md

```diff
@@ -35,20 +36,25 @@ accessory drawings in this document are for illustration only, and may not exact
 
 ## Adding Switch Admin Password to Vault
 
-If CSM has been installed and Vault is running, add the switch credentials into Vault.
-Certain tests, including `goss-switch-bgp-neighbor-aruba-or-mellanox` use these credentials to test the state of the switch.
-This step is not required to configure the management network.
-If Vault is unavailable, this step can be temporarily skipped.
-Any automated tests that depend on the switch credentials being in Vault will fail until they are added.
+<!-- markdownlint-disable-next-line MD013 MD026 -->
+If CSM has been installed and Vault is running, add the switch credentials into Vault. Certain tests, including `goss-switch-bgp-neighbor-aruba-or-mellanox` use these credentials to test the state of the switch. This step is not required to configure the management network. If Vault is unavailable, this step can be temporarily skipped. Any automated tests that depend on the switch credentials being in Vault will fail until they are added.
```

Comment:
Instead of ignoring the linter here, I'd strongly recommend fixing the line.
If someone views this with a terminal, the line is atrociously long as-is.